apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -26,9 +26,48 @@
 
 class SparseNDArray(NDArray, somacore.SparseNDArray):
     """
-    Represents ``X`` and others.
+    ``SparseNDArray`` is a sparse, N-dimensional array, with offset (zero-based)
+    integer indexing on each dimension. ``SparseNDArray`` has a user-defined
+    schema, which includes:
+    - the element type, expressed as an Arrow type, indicating the type of data
+      contained within the array, and
+    - shape - the shape of the array, i.e., number and length of each dimension
```

Comment:
my #1 preference is

```
- shape: the shape of ...
```

with #2 preferece being

```
- shape -- the shape of
```

a single `-` belongs between words, as in _N-dimensional_